### ISSUE TITLE
Database shape

### ISSUE THREAD

**Issue by piratuks** [visjs/vis-network#403]
Regarding shape type database. The last thing i noticed from the new feature is corner angle for database shape. 
![Capture](https://user-images.githubusercontent.com/8760197/73301784-1e1dac00-421c-11ea-8938-b99ed3d749e5.PNG)

My guess is because top and bot is rounded - distanceToNode is not enough for the calculation of self circle position based on angle? 

   var nodes = new vis.DataSet([
            {
                id: 3,
                label:
                    ["Self reference edge is in the bottom left corner",
                        "with parts rendered behind node"].join("\n"),
                widthConstraint: { minimum: 250 }, heightConstraint: { minimum: 200 },
                x: 400,
                y: 400,
                color: {
                    background: "transparent"
                },
                shape: "database"
            },
        ]);

        // create an array with edges
        var edges = new vis.DataSet([
            {
                from: 3, to: 3, label: "", arrows: 'to, middle, from',     
            }
        ]);

        // create a network
        var container = document.getElementById("mynetwork");
        var data = {
            nodes: nodes,
            edges: edges
        };
        var options = {
            edges: {
                color: {
                    color: "#2B7CE9",
                    highlight: "#2B7CE9",
                    hover: "#2B7CE9"
                },
                hoverWidth: function (width) { return width + 2; },
                shadow: true
            },
            interaction: {
                hover: true,
            },
        };
        var network = new vis.Network(container, data, options);

**Comment by Thomaash** [comment]
Yeah, I know about this. Not all shapes implement this properly. Do you plan to work on this one too?

**Comment by piratuks** [comment]
Yea most likely

**Comment by Thomaash** [comment]
Thanks.
